STPConfig.cmake.in

```diff
@@ -13,6 +13,7 @@ set(STP_INCLUDE_DIRS "@CONF_INCLUDE_DIRS@")
 
 # Our library dependencies (contains definitions for IMPORTED targets)
 include("${STP_CMAKE_DIR}/@STP_TARGETS_FILENAME@")
+find_package(cryptominisat5 CONFIG)
```

Comment:
Not sure if that matters, but how about including it before that `include` statement?

Reply:
Fixed now :)